pages/docs/tutorials/native/basic-kotlin-native-app.md

```diff
@@ -42,26 +42,24 @@ fun main(args: Array<String>) {
 
 ## Compiling and examining output 
 
-The next step is to compile the application, which will be done using the compiler downloaded in the first step. If we have the `bin` folder
-correctly added to the path, we should be able to invoke the compiler using
+The Kotlin compiler uses a technology known as [LLVM](https://en.wikipedia.org/wiki/LLVM) to target multiple platforms. LLVM requires as input what's know as intermediate representation or IR. This IR is
+represented by a bitcode file, which is a bitstream file format. 
 
-`kotlinc hello.kt`
+![Compiler Diagram]({{ url_for('tutorial_img', filename='native/first-kotlin-native/llvm-diagram.png')}})
 
-which is telling the compiler to compile the source code `hello.kt`.
 
-The first time the compiler runs, it downloads a list of necessary requirements (thus first run does take longer). If everything runs correctly, the output 
-should be two files  
+We now need to compile our application, which is done using the compiler downloaded in the first step. If we have the `bin` folder
+correctly added to the path, we should be able to invoke the compiler using
 
-`hello-world.kt.bc`
-`hello.kexe`
+`kotlinc hello.kt`
 
-The filename with extension `bc` is what's known a a `bitcode` file. To understand what this is, we first need to see how Kotlin/Native works.
- 
-The Kotlin compiler uses a technology known as [LLVM](https://en.wikipedia.org/wiki/LLVM) to target multiple platforms. LLVM requires as input what's know as intermediate representation or IR. This IR is
-represented by the bitcode file, which is a bitstream file format. 
+which is telling the compiler to compile the source code `hello.kt`.
 
-![Compiler Diagram]({{ url_for('tutorial_img', filename='native/first-kotlin-native/llvm-diagram.png')}})
+The first time the compiler runs, it downloads a list of necessary requirements (thus first run does take longer). 
+While there are no runtime dependencies for binaries produced by Kotlin/Native, the compiler itself does require Java 8. If everything runs correctly, the output 
```

Comment:
The current wording leads users to assume that the compiler will download Java 8 by itself, which is not the case. You should make it clear (most likely earlier in the tutorial) that Java 8 needs to be installed.